packages/fxa-settings/src/pages/AccountRecoveryResetPassword/en.ftl

```diff
@@ -0,0 +1,8 @@
+password-link-expired = Reset password link expired
+password-link-damaged = Reset password link damaged
```

Comment:
```suggestion
# Appears when a link to reset password is damaged
password-link-damaged-header = Reset password link damaged
```

Reply:
Individual level comment and tweak to ID to provide context for localizers. "Reset password link damaged" may be hard to parse without context.